crates/router/src/core/payments/operations.rs

```diff
@@ -75,6 +77,18 @@ pub struct ValidateResult<'a> {
     pub requeue: bool,
 }
 
+#[cfg(feature = "locking")]
+impl locking::GetLockingInput for ValidateResult<'_> {
+    fn get_locking_input(&self) -> RouterResult<locking::LockingInput> {
+        Ok(locking::LockingInput {
+            unique_locking_key: self.payment_id.get_id(),
+            api_identifier: "PAYMENTS".to_string(),
```

Comment:
will prefer if this is a constant, defined somewhere

Reply:
Yep will remove all hardcodings. Had already added in TODO.